modules/Microsoft.CDN/profiles/endpoints/origins/deploy.bicep

```
@@ -19,11 +19,23 @@ param httpPort int = 80
 @description('Optional. The HTTPS port of the origin.')
 param httpsPort int = 443
 
-@description('Optional. The priority of origin in given origin group for load balancing.')
-param priority int = 1
+@description('Conditional. The priority of origin in given origin group for load balancing, weight should also be provided.')
```

```suggestion
@description('Conditional. The priority of origin in given origin group for load balancing. Required if `weight` is provided.')
```

Just to align the teminology with the other 'conditional' parameters.